deploy/examples/radosnamesapce-mirrored.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPoolRadosNamespace
+metadata:
+  name: namespace-a
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # The name of the RADOS namespace. If not set, the default is the name of the CR.
+  # name: namespace-a
+  # blockPoolName is the name of the CephBlockPool CR where the namespace will be created.
+  blockPoolName: replicapool
+  mirroring:
+    enabled: false
+    remoteNamespace: namespace-a
+    # mirroring mode: pool level or per image
+    # for more details see: https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#enable-mirroring
+    mode: image
+    # specify the schedule(s) on which snapshots should be taken
+    # snapshotSchedules:
+    #   - interval: 24h # daily snapshots
```

Comment:
What is the default snapshot schedule? Is it daily like this example? If so, how about uncommenting it? Even though it's the default, it's more readable that way in the example.

Reply:
Not sure about the default